### PR TITLE
[EDU-853] - Correct token revoc error handling code

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Corrects the error handling code in the two code samples:

* [EDU-853](https://ably.atlassian.net/browse/EDU-853).

## Review

Check the two code samples, ideally under web view.

* [Token revoc code](https://ably-docs-edu-853-fix-c-9gp7ti.herokuapp.com/core-features/authentication/#token-revocation)

Note under web view keys are populated automatically.

### CODE REVIEW

File: content/core-features/authentication.textile [hunk 1]
```diff
@@ -468,33 +468,43 @@ h4. Examples
...
 ```
 
 A sample request to revoke a JWT based on @revocationKey@, using the REST client library SDK:
 
 ```[javascript]
+const Ably = require('ably');
+
 const ablyRest = new Ably.Rest({ key: '{{API_KEY}}' })
-const requestBody = {"targets": [ "revocationKey:client1@example.com" ]}                                                                                                                                                                                       
+const requestBody = { targets: ['revocationKey:client1@example.com'] }
```
Comment (marklewin) [inline content/core-features/authentication.textile:495]: @tbedford I'm just wondering whether this should look more like a Key than just re-using the clientId from the preceding example. WDYT?

Comment (marklewin) [reply content/core-features/authentication.textile:495]: revoc

Comment (tbedford) [reply content/core-features/authentication.textile:495]: Good point. `revocationKey` is an arbitrary string, but at the moment it's a bit confusing to use the same string in both examples.

**Review by marklewin — APPROVED** [review]
LGTM

**Pull request merged**